### PR TITLE
[terra-action-header] Remove Terra.it helpers from wdio tests.

### PR DESCRIPTION
**Pull request by pranav300**
### Summary

Remove Terra.it helpers from terra-action-header's wdio tests since they are now deprecated.

### CODE REVIEW

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/wrapped-title-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('wrapped title');
   });
 
-  describe('End Element Only', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/end-element-action-header'));
+  it('displays ActionHeader with end element only', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/end-element-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:63]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/end-element-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/previous-enabled-next-disabled-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('previous enabled next disabled');
   });
 
-  describe('Wrapped Title', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/wrapped-title-action-header'));
+  it('displays ActionHeader with wrapped title', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/wrapped-title-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:57]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/wrapped-title-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/next-enabled-previous-disabled-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('next enabled previous disabled');
   });
 
-  describe('Previous Enabled Next Disabled', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/previous-enabled-next-disabled-action-header'));
+  it('displays ActionHeader with previous enabled next disabled', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/previous-enabled-next-disabled-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:51]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/previous-enabled-next-disabled-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 4]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/back-close-previous-next-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('back close previous next');
   });
 
-  describe('Next Enabled Previous Disabled', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/next-enabled-previous-disabled-action-header'));
+  it('displays ActionHeader with next enabled previous disabled', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/next-enabled-previous-disabled-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:45]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/next-enabled-previous-disabled-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 5]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/maximize-close-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('maximize close');
   });
 
-  describe('Back Close Previous Next', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/back-close-previous-next-action-header'));
+  it('displays ActionHeader with back close previous next', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/back-close-previous-next-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:39]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/back-close-previous-next-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 6]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/minimize-custom-button-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('minimize custom button');
   });
 
-  describe('Maximize Close', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/maximize-close-action-header'));
+  it('displays ActionHeader with maximize close', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/maximize-close-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:33]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/maximize-close-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 7]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/close-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('close');
   });
 
-  describe('Minimize Custom Button', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/minimize-custom-button-action-header'));
+  it('displays ActionHeader with minimize custom button', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/minimize-custom-button-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:27]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/minimize-custom-button-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 8]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/#/raw/tests/terra-action-header/action-header/back-close-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('back close ');
   });
 
-  describe('Close', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/close-action-header'));
+  it('displays ActionHeader with close', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/close-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:21]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/close-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 9]
```diff
@@ -1,67 +1,67 @@
...
     before(() => browser.url('/#/raw/tests/terra-action-header/action-header/back-action-header'));
 
-    Terra.it.validatesElement();
+    Terra.validates.element('back');
   });
 
-  describe('Back Close', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/back-close-action-header'));
+  it('displays ActionHeader with back and close', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/back-close-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:15]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/back-close-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 10]
```diff
@@ -1,67 +1,67 @@
...
+  it('Displays default ActionHeader', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/default-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('default');
   });
 
-  describe('Back', () => {
+  it('displays ActionHeader with back', () => {
     before(() => browser.url('/#/raw/tests/terra-action-header/action-header/back-action-header'));
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:9]:
```suggestion
    before(() => browser.url('/raw/tests/terra-action-header/action-header/back-action-header'));
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 11]
```diff
@@ -1,67 +1,67 @@
 Terra.describeViewports('ActionHeader', ['tiny', 'medium', 'large'], () => {
-  describe('Default', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/default-action-header'));
+  it('Displays default ActionHeader', () => {
+    browser.url('/#/raw/tests/terra-action-header/action-header/default-action-header');
```
Comment (mjhenkes) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:3]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/default-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 12]
```diff
@@ -1,67 +1,67 @@
...
+    browser.url('/raw/tests/terra-action-header/action-header/default-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('default');
   });
 
-  describe('Back', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/back-action-header'));
+  it('displays ActionHeader with back', () => {
+    before(() => browser.url('/raw/tests/terra-action-header/action-header/back-action-header'));
```
Comment (StephenEsser) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:9]:
```suggestion
    browser.url('/raw/tests/terra-action-header/action-header/back-action-header');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js [hunk 13]
```diff
@@ -1,67 +1,67 @@
...
+    Terra.validates.element('back');
   });
 
-  describe('Back Close', () => {
-    before(() => browser.url('/#/raw/tests/terra-action-header/action-header/back-close-action-header'));
+  it('displays ActionHeader with back and close', () => {
+    browser.url('/raw/tests/terra-action-header/action-header/back-close-action-header');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('back close ');
```
Comment (StephenEsser) [inline packages/terra-action-header/tests/wdio/action-header-spec.js:17]:
```suggestion
    Terra.validates.element('back close');
```

File: packages/terra-action-header/tests/wdio/action-header-spec.js (hunk 12 above)
Comment (pranav300) [reply packages/terra-action-header/tests/wdio/action-header-spec.js]: Updated here https://github.com/cerner/terra-core/pull/3218/commits/d45de48c89853bbd258d48246ccc37f7df66f3fc

File: packages/terra-action-header/tests/wdio/action-header-spec.js (hunk 13 above)
Comment (pranav300) [reply packages/terra-action-header/tests/wdio/action-header-spec.js]: Updated here https://github.com/cerner/terra-core/pull/3218/commits/d45de48c89853bbd258d48246ccc37f7df66f3fc

**Review by emilyrohrbough — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Pull request merged**